autoRSA.py

```diff
@@ -572,6 +577,11 @@ async def rsaadd(ctx, broker=None, credentials=None):
                         raise Exception(
                             "Invalid credentials. Just enter your email for Fennel."
                         )
+                elif broker =="Dspac":
+                    if len(credentials.split(":")) != 1:

```

Comment:
this check is also incorrect. after splitting, it should be 2 not 1, since it's username and password creds

Reply:
updated